docs/features/W3cCredentials.md

```diff
@@ -0,0 +1,633 @@
+## Verifiable Credential Data Integrity (VC-DI) Credentials in Aries Cloud Agent Python (ACA-Py)
+
+This document outlines a new functionality within Aries Agent that facilitates the issuance of credentials and presentations in compliance with the W3C standard.
+
+### Table of Contents
+
+- [General Concept](#general-concept)
+- [Prerequisites](#prerequisites)
+  - [Verifiable Credentials Data Model](#verifiable-credentials-data-model)
+  - [Verifiable Presentations Data Model](#verifiable-presentations-data-model)
+  - [DIF Presentation Format](#dif-presentation-format)
+- [Preparing to Issue a Credential](#preparing-to-issue-a-credential)
+  - [VC-DI Context](#vc-di-context)
+  - [Signature Suite](#signature-suite)
+  - [DID Method](#did-method)
+- [Issue a Credential](#issue-a-credential)
+- [Verify a Credential](#verify-a-credential)
+- [Present Proof](#present-proof)
+  - [Requesting Proof](#requesting-proof)
+  - [Presenting Proof](#presenting-proof)
+  - [Verifying Proof](#verifying-proof)
+- [Appendix](#appendix)
+  - [Glossary of Terms](#glossary-of-terms)
+  - [References and Resources](#references-and-resources)
+
+### General Concept
+
+The introduction of VC-DI credentials in ACA-Py facilitates the issuance of credentials and presentations in adherence to the W3C standard.
+
+### Prerequisites
+
+Before utilizing this feature, it is essential to have the following:
+
+#### Verifiable Credentials Data Model
+
+A basic understanding of the Verifiable Credentials Data Model is required. Resources for reference include:
+
+- [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/)
+
+#### Verifiable Presentations Data Model
+
+Familiarity with the Verifiable Presentations Data Model is necessary. Relevant resources can be found at:
+
+- [Verifiable Presentations Data Model](https://www.w3.org/TR/vc-data-model/#presentations)
+
+#### DIF Presentation Format
+
+Understanding the DIF Presentation Format is recommended. Access resources at:
+
+- [DIF Presentation Format](https://identity.foundation/presentation-exchange/)
+
+### Preparing to Issue a Credential
+
+To prepare for credential issuance, the following steps must be taken:
+
+#### VC-DI Context
+
+Ensure that every property key in the document is mappable to an IRI. This requires either the property key to be an IRI by default or to have the shorthand property mapped in the `@context` of the document.
+
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/data-integrity/v2",
+        {
+            "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+        }
```

Comment:
I would include note about avoiding `@vocab` outside of development and/or examples, and implementers should define their own terms in a context file they provide.

Reply:
For this specific example, { "name": "https://schema.org/name" } instead of {"@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"} would also be a better solution to defining the attribute.

Reply:
do you mean something like this
```json
..
"@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/security/data-integrity/v2",
    "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
    ]
 ..
 ```

Reply:
no, leave as is this will work, I would just put a note since using `@vocab` is considered a bad practice outside of demonstrations and development. Implementers are encouraged to write their own context or use community available contexts. The existing w3c documentation outlines this here:
https://github.com/hyperledger/aries-cloudagent-python/blob/main/docs/features/JsonLdCredentials.md#json-ld-context